ctapipe/reco/__init__.py

```diff
@@ -1,13 +1,12 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-from .reco_algorithms import Reconstructor
-from .hillas_reconstructor import HillasReconstructor
-from .impact import ImPACTReconstructor
+from .hillas_reconstructor import HillasReconstructor, Reconstructor
```

Comment:
Please import the `Reconstructor` from the place where it is defined, not some other module where it is used. It was good before.

Reply:
done